src/nvim/ex_docmd.c

```diff
@@ -6816,7 +6840,7 @@ void post_chdir(int local)
 
 
 /*
- * ":cd", ":lcd", ":chdir" and ":lchdir".
+ * ":cd", ":tcd", ":lcd", ":chdir", ":tchdir" and ":lchdir".
```

Comment:
Probably because it's in the style of the original comment, clint doesn't complain about Vim style. Fixed.